### PR TITLE
Update changelog after upgrade to faraday 2

### PR DESCRIPTION
**Pull request by kirillkaiumov**
Relates to https://github.com/ashkan18/graphlient/pull/95

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-### 0.6.1 (Next)
+### 0.7.0 (Next)
```
Comment (dblock) [inline CHANGELOG.md:1]: The version in https://github.com/ashkan18/graphlient/blob/master/lib/graphlient/version.rb should be incremented with this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md:3]: Put this back :)

**Review by dblock — CHANGES_REQUESTED** [review]
Fix danger please?

**Comment by ashkan18** [comment]
thanks @kirillkaiumov , I resolved the conflicts which addresses @dblock 's concern as well. I'll merge once green.

**Pull request merged**